From 22829adb047b16e8b26d8e5d425bd0d892434593 Mon Sep 17 00:00:00 2001
From: Rong Dian <b38775@freescale.com>
Date: Mon, 27 Aug 2012 11:27:06 +0800
Subject: ENGR00221450 imx6 thermal: add sys_close() in cooling device

add sys_close() to close opened file in cooling device

Signed-off-by: Rong Dian <b38775@freescale.com>
---
 drivers/mxc/thermal/cooling.c |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/mxc/thermal/cooling.c b/drivers/mxc/thermal/cooling.c
index 31188f0..7019d99 100644
--- a/drivers/mxc/thermal/cooling.c
+++ b/drivers/mxc/thermal/cooling.c
@@ -178,6 +178,7 @@ int anatop_thermal_cpu_hotplug(bool cpu_on)
 					sys_write(fd, (char *)"1", MAX_CPU_ONLINE_LEN);
 					cpu_mask &= ~(0x1 << cpu);
 					ret = 0;
+					sys_close(fd);
 					break;
 				}
 				sys_close(fd);
@@ -199,6 +200,7 @@ int anatop_thermal_cpu_hotplug(bool cpu_on)
 					sys_write(fd, (char *)"0", MAX_CPU_ONLINE_LEN);
 					cpu_mask |= 0x1 << cpu;
 					ret = 0;
+					sys_close(fd);
 					break;
 				}
 				sys_close(fd);
-- 
1.7.9.5

